### ISSUE TITLE
Failure of setSpeciesFillColor

### ISSUE THREAD

**Issue by luciansmith** [sys-bio/SBMLNetwork#30]
This code returns '-1' for setSpeciesFillColor, for no obvious reason.
```c++
      LIBSBMLNETWORK_CPP_NAMESPACE::autorender(&m_sbml, m_autolayout.maxNumConnectedEdges);
      int ret = LIBSBMLNETWORK_CPP_NAMESPACE::setSpeciesFillColor(&m_sbml, 0, "#a0e0a030");
```

**Comment by luciansmith** [comment]
Same for setCompartmentStrokeColor:

```c++
                ret = LIBSBMLNETWORK_CPP_NAMESPACE::setCompartmentStrokeColor(&m_sbml, 0, "#e69600b0");
```

**Comment by luciansmith** [comment]
Same for setReactionStrokeColor:

```c++
    ret = LIBSBMLNETWORK_CPP_NAMESPACE::setReactionStrokeColor(&m_sbml, 0, "#000000a0");
```

**Comment by adelhpour** [comment]
I could reproduce the bug. I'll fix it!

**Comment by adelhpour** [comment]
This is fixed with PR #32
